ui/desktop/config/environment.js

```diff
@@ -61,9 +61,7 @@ module.exports = function (environment) {
     },
 
     featureFlags: {
-      search: false,
-      filter: true,
-      'post-connection-workflow': false,
```

Comment:
I didn't see these flags used anywhere, does anyone have any context for these?

Reply:
It looks like I added that but we ended up putting all of that work into an llb and we never used the flag. Thanks for cleaning that up.